crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -885,8 +890,12 @@ type GenesisTransaction {
 }
 
 
+"""
+An immutable object is an object that can't be mutated, transferred, or deleted.
+Immutable objects have no owner, so anyone can use them.
```

Comment:
Since immutable objects cannot be mutated, transferred, or deleted, the only thing we can do is read them right? But all objects are readable, so not sure if this usability is unique to immutable objects